lib/logflare/alerting.ex

```diff
@@ -150,15 +155,7 @@ defmodule Logflare.Alerting do
   def get_alert_job(%AlertQuery{id: id}), do: get_alert_job(id)
 
   def get_alert_job(id) do
-    AlertsScheduler.jobs()
-    |> Enum.find_value(fn {_ref,
-                           %Quantum.Job{
-                             task: {_module, _function, [%AlertQuery{id: query_id}, _]}
-                           } = job} ->
-      if query_id == id do
-        job
-      end
-    end)
+    remote_call_scheduler(:find_job, ["#{id}"])
```

Comment:
```suggestion
    remote_call_scheduler(:find_job, [Integer.to_string(id)])
```

(doesn't involve protocol dispatch and is more explicit)